RogueMaster/flipperzero-firmware-wPlugins
# Tomodachi FAP and Tomodachi SRV

# What's new

- Tomodachi FAP and Tomodachi SRV

# Verification 

- Have two or more FlipperZero devices
- Run FAP on at least one FZ
- Wait at least 6 minutes
- Hear beep + vibe
- Check screen, name of detected FZ will appear

# Checklist (For Reviewer)
